etl-pipeline/model/postgres/record.py

```diff
@@ -114,6 +114,15 @@ class Record(Base, Core):
         index=True
     )
     cluster_status = Column(Boolean, default=False, nullable=False, index=True)
+    state = Column(
+        Unicode,
+        ENUM(
+            "ingested", "files_saved", "complete",
+            name="record_state", create_type=False,
+        ),
+        nullable=True,
+        index=True,
```

Comment:
i think in the past we have failed to create indices on the database because there's too much going on :| 

what are your thoughts on not making this an index to start and then adding the index later?